feat: add waffle flag to enable multiple LTI 1.3 launch URLs (#557)
openedx/xblock-lti-consumer: README.rst

4. Create a new external LTI configuration and use it in the XBlock.
   This is explained in the README of the `openedx-ltistore`_ repository.

> 5. (Optional) To allow overriding the `lti_1p3_launch_url` per block, create another override:

Reviewer: ```suggestion:
5. (Optional) Allows overriding of the `lti_1p3_launch_url` per block:
```
